packages/wdio-local-runner/src/index.ts

```diff
@@ -3,16 +3,24 @@ import { WritableStreamBuffer } from 'stream-buffers'
 
 import WorkerInstance from './worker'
 import { SHUTDOWN_TIMEOUT, BUFFER_OPTIONS } from './constants'
+import type { WorkerRunPayload } from './types'
 
 const log = logger('@wdio/local-runner')
 
+interface RunArgs extends WorkerRunPayload {
+    command: string
+    args: any
+}
+
 export default class LocalRunner {
-    constructor (configFile, config) {
-        this.configFile = configFile
+    config: WebdriverIO.Config
+    workerPool: Record<string, WorkerInstance> = {}
+
+    stdout = new WritableStreamBuffer(BUFFER_OPTIONS)
+    stderr = new WritableStreamBuffer(BUFFER_OPTIONS)
+
+    constructor (configFile: string, config: WebdriverIO.Config) {
```

Comment:
Just noticed that previously we assigned the configFile which is now missing, is this an oopsy?

Reply:
Last time I forgot you can also do:
```suggestion
    constructor (public configFile: string, public config: WebdriverIO.Config) {
```
Which allows you to remove the declarations above for these two

Reply:
> is this an oopsy?

no, it was there before but not used in the local runner (I think it was in the lambda one). Happy to clean this up eventually as part of a separate PR.